prolog/terminus_store.pl

```diff
@@ -703,6 +713,15 @@
     create_named_graph(X, "sometestdb", _),
     open_named_graph(X, "sometestdb", _).
 
+test(delete_named_graph_memory) :-
+    open_memory_store(X),
+    create_named_graph(X, "sometestdb", _),
+    delete_named_graph(X, "sometestdb").
```

Comment:
you should verify here that the graph is in fact gone after your delete.

open_named_graph will fail if the graph is not there, so you can check for that with
`\+ open_named_graph(X, "sometestdb", _).`